encryption.rst

```diff
@@ -113,6 +121,74 @@ like so.
    $ sudo -E {command} build --encrypt encrypted.sif encrypted.def
    Starting build...
 
+Image info verification (``Privileged Encryption``)
+---------------------------------------------------
+
+.. code:: 
+
+   $ {command} sif list encrypted.sif
+
+Example:
+
+.. code:: 
+
+   ------------------------------------------------------------------------------
+   ID   |GROUP   |LINK    |SIF POSITION (start-end)  |TYPE
+   ------------------------------------------------------------------------------
+   1    |1       |NONE    |32176-32223               |Def.FILE
+   2    |1       |NONE    |32223-34202               |JSON.Generic
+   3    |1       |NONE    |34202-34292               |JSON.Generic
+   4    |1       |NONE    |36864-17608704            |FS (Encrypted squashfs/*System/arm64)
+   5    |1       |4       |17608704-17609449         |Cryptographic Message (PEM/RSA-OAEP)
+
+Note that ``ID=4`` partition has the type of ``Encrypted squashfs``
+
+Encrypting with a passphrase interactively (``Unprivileged Encryption``)
+------------------------------------------------------------------------
+
+.. code::
+
+   $ {command} build --passphrase encrypted.sif encrypted.def
+   Enter encryption passphrase: <secret>
+   INFO:    Starting build...
+
+Using an environment variable (``Unprivileged Encryption``)
+-----------------------------------------------------------
+
+.. code::
+
+   $ {ENVPREFIX}_ENCRYPTION_PASSPHRASE=<secret> {command} build encrypted.sif encrypted.def
+   Starting build...
+
+
+.. code::
+
+   $ export {ENVPREFIX}_ENCRYPTION_PASSPHRASE=$(cat secret.txt)
+
+   $ {command} build encrypted.sif encrypted.def
+   Starting build...
+
+Image info verification (``Unprivileged Encryption``)
+-----------------------------------------------------
+
+.. code:: 
+
+   $ {command} sif list encrypted.sif
+
+Example:
+
+.. code:: 
+
+   ------------------------------------------------------------------------------
+   ID   |GROUP   |LINK    |SIF POSITION (start-end)  |TYPE
+   ------------------------------------------------------------------------------
+   1    |1       |NONE    |32176-32215               |Def.FILE
+   2    |1       |NONE    |32215-36269               |JSON.Generic
+   3    |1       |NONE    |36269-36465               |JSON.Generic
+   4    |1       |NONE    |36864-26386432            |FS (Gocryptfs squashfs/*System/arm64)
+
+Note that ``ID=4`` partition has the type of ``Gocryptfs squashfs``
```

Comment:
And here use
```
Note that partition 4 is type ``Gocryptfs squashfs``.
```

Reply:
thanks for the comments
https://github.com/apptainer/apptainer-userdocs/pull/179/commits/816c972aee9ca359516da01997481793ca62c2ca